File: src/main/java/dominio/indicadores/ExpresionCuenta.java
package dominio.indicadores;

import dominio.empresas.Empresa;

import java.time.Year;

public class ExpresionCuenta implements Expresion{
	
	private String cuenta;
	
	public ExpresionCuenta(String cuenta) {
		this.cuenta = cuenta;
	}

	public int evaluarEn(Empresa empresa, Year anio) {
		return empresa.getValorCuenta(cuenta,anio);
	}
	
}

File: src/main/java/dominio/indicadores/ExpresionCuenta.kt
package dominio.indicadores

import dominio.empresas.Empresa

import java.time.Year

class ExpresionCuenta(private val cuenta: String) : Expresion {

    override fun evaluarEn(empresa: Empresa, anio: Year): Int = empresa.getValorCuenta(cuenta, anio)

}
